stream-chat-android-ui-components-sample/src/main/res/values/themes.xml

```diff
@@ -63,5 +63,7 @@
 
     <style name="SearchInputViewTheme" parent="StreamUi.SearchInputView"/>
 
+    <style name="MediaActivityStyle" parent="StreamUi.MediaActivity"/>
```

Comment:
Last thing - we don't need those styles in sample app know, right?

Reply:
We don't, but they are an interesting way to test the themes or to show how to configure an app incase someone wants to read our code

Reply:
Nevermind... The styles for activities are not used, only the ones of Views. Let me remove this.